dashboard/admin.py

```diff
@@ -48,7 +48,7 @@ def clean(self):
 class CourseAdmin(admin.ModelAdmin):
     inlines = [CourseViewOptionInline, ]
     form = CourseForm
-    list_display = ('id', 'canvas_id', 'name', 'term', '_courseviewoption')
+    list_display = ('canvas_id', 'name', 'ab_test_course', 'term', '_courseviewoption')
```

Comment:
Removed the Canvas long id from the admin display list as it was not needed

Reply:
This is still showing up in the Django Admin, I think. I believe we also need to remove `id` from the `readonly_fields`.

Reply:
Hm, scratch that. When you remove `id` from `readonly`, it starts showing up as an editable field. There must be another way to remove it entirely...

Reply:
Yeah, I think the way to do it would be to make `id` on the `Course` model `editable=False` and then run a migration. Then you can remove `id` from `readonly` and it doesn't show up.

Reply:
Thanks @ssciolla . Removed the long canvas id from the Django admin view